session-3-jshell/README.md

```diff
@@ -18,15 +43,18 @@ And locate the command from the above list. Ensure `JAVA_HOME/bin` is in your `P
 
     $ echo $PATH
     /usr/lib/jvm/jdk-9/bin/:....
+
+    Note: If JAVA_HOME is not setting on you Mac or Linux (Or it's no a Java 9 version) or PATH doesn't show the bin folder for JDK-9, You should set the correct value following next steps:(For Mac Users the JDK 9 could be on a path as /Library/Java/JavaVirtualMachines/jdk9/Contents/Home)
```

Comment:
If `JAVA_HOME` is not set on your Mac or Linux machine (or it's not set to Java 9) or `PATH` doesn't show the `bin` folder for JDK-9, You should set the correct values following the next steps:
(For Mac Users the JDK 9 path could be `/Library/Java/JavaVirtualMachines/jdk9/Contents/Home`)